client/src/components/BulkCreateForm/BulkCreateForm.js

```diff
@@ -507,6 +515,35 @@ export default class BulkCreateForm extends React.Component {
 
         results = results.filter(result => result.errors.length > 0 || result.warnings.length > 0);
 
+        /**
+         * Get the tooltip message based on the error/warning message
+         *
+         * @param msg error/warning message
+         * @returns {string} the tooltip message
+         */
+        const getTooltip = (msg) => {
+            // console.log(msg)
+            switch (msg) {
+                case 'Address OR Coordinates are required':
+                    return 'Valid coordinate formats are:\n' +
+                        '43.084670, -77.674357\n' +
+                        '43°05\'04.8", -77°40\'27.7"\n' +
+                        '43°05\'04.8"N, 77°40\'27.7"W'
+            }
+        }
+
+        /**
+         * Helper function for composing the tooltip
+         *
+         * @param props OverlayTrigger properties
+         * @returns {JSX.Element} the Tooltip
+         */
+        const renderTooltip = (props) => (
```

Comment:
When compiling the front-end, there are a few warnings about variables that are defined but never used. The easiest way to rectify this would just be commenting any tool tip code out and putting a TODO in somewhere saying to fix it at some point. Alternatively, we could make a separate branch for it. 
![image](https://user-images.githubusercontent.com/35973124/137197169-ea56ba34-8dd2-460e-9f57-450c23e69f55.png)